src/react-components/codebox.tsx

```diff
@@ -27,12 +29,13 @@ export const CodeBox = ({
       {...props}
     >
       <input id={id} type="checkbox" />
-      <div
+      <label
+        htmlFor={id}
```

Comment:
This makes the whole header clickable

https://github.com/ory/elements/assets/5540255/5ebef1e5-dd0d-4699-8916-c7be30f48cd3